woodwork/type_sys/inference_functions.py

```diff
@@ -115,17 +115,10 @@ def boolean_nullable_func(series):
         series_no_null = series.dropna()
         try:
             series_no_null_unq = set(series_no_null)
```

Comment:
Should we apply the `.unique()` suggestion here as well?

Reply:
I decided not to do that because `.unique` only really provides a speed improvement for series with an `integer` dtype. I could put it in there for consistency though